values.yaml

```diff
@@ -73,7 +73,7 @@ properties:
   isNode: false # set to false if ldap is enabled
   httpPort: null # set to null if ldap is enabled
   httpsPort: 9443 # set to 9443 if ldap is enabled
-  webProxyHost:
+  webProxyHost: <minikube_IP>:30236
```

Comment:
Add comment here with short explanation on how to get the minikube ip and what should be there when not using minikube, with sample values.

Add an entry for this config value in the README.md with that information.